ck+/script/data.js

```diff
@@ -218,6 +218,12 @@ var doubleHitMoves = new Set([
 	"bonemerang", "double-hit", "double-kick", "twineedle"
 ]);
 
+//ADD GENERAL FLAGS OBJECT TO GLOBAL VARIABLES
+const game_general_flags = {
```

Comment:
Might be better to go with a shorter name? I imagine flags will be used all over the codebase so something like `flags` would probably same some time.

Reply:
Got it, I tend to use longer names to avoid comments but, in this case, I used both of them. I'll try to use shorter names from now on.